modules/distribution/src/repository/resources/dbscripts/apimgt/h2.sql

```diff
@@ -196,7 +198,7 @@ CREATE TABLE IDN_AUTH_SESSION_STORE (
             SESSION_OBJECT BLOB,
             TIME_CREATED BIGINT,
             TENANT_ID INTEGER DEFAULT -1,
-                    PRIMARY KEY (SESSION_ID, SESSION_TYPE, TIME_CREATED, OPERATION)
+		        PRIMARY KEY (SESSION_ID, SESSION_TYPE, TIME_CREATED, OPERATION)
```

Comment:
Directly copied content from https://github.com/wso2/carbon-apimgt/tree/v5.0.4/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql as apim version upgraded to v5.04